include/mlir/IR/OpImplementation.h

```diff
@@ -594,6 +594,9 @@ class OpAsmDialectInterface
   /// Get a special name to use when printing the given operation. The desired
   /// name should be streamed into 'os'.
   virtual void getOpResultName(Operation *op, raw_ostream &os) const {}
+  /// Get a special name to use when printing the block arguments. The desired
```

Comment:
This should be updated specifically mention that these are only entry block arguments.